04.ls/ls.rb

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+COLUMN_COUNT = 3
+
+def main
+  filenames = Dir.glob('*')
+  show_file_list(filenames)
+end
+
+def show_file_list(filenames)
+  maximum_length = filenames.max_by(&:length).length
```

Comment:
```suggestion
  max_length = filenames.max_by(&:length).length
```

max_byもmaxなので、ここもmaxでいいかもしれません